node/package.json

```diff
@@ -25,6 +25,7 @@
     "prepare": "cd .. && husky install"
   },
   "dependencies": {
+    "@kubernetes/client-node": "^0.18.1",
```

Comment:
with restructure, we should only need this dependency declared long term

Reply:
Once we restructure yeah, sadly now it needs to stay